### PR TITLE
Remove CUDA and OFED builds from CI

### PR DESCRIPTION
**Pull request by bertiethorpe**

### CODE REVIEW

**Comment by bertiethorpe** [comment]
Nightly build:
https://github.com/stackhpc/ansible-slurm-appliance/actions/runs/11854169207

File: .github/workflows/nightlybuild.yml [hunk 1]
```diff
@@ -42,6 +35,7 @@ jobs:
           "RL8": "Rocky-8-GenericCloud-Base-8.9-20231119.0.x86_64.qcow2",
           "RL9": "Rocky-9-GenericCloud-Base-9.4-20240523.0.x86_64.qcow2"
         }
+      BUILD: openstack.openhpc
```
Comment (sjpb) [inline .github/workflows/nightlybuild.yml:38]: This isn't right, should be `rocky-latest`

**Review by sjpb — CHANGES_REQUESTED** [review]

**Review by sjpb — APPROVED** [review]
LGTM

**Comment by bertiethorpe** [comment]
Nightly build:
https://github.com/stackhpc/ansible-slurm-appliance/actions/runs/11855609687

**Comment by bertiethorpe** [comment]
Fat image build:
https://github.com/stackhpc/ansible-slurm-appliance/actions/runs/11855970106

**Review by wtripp180901 — APPROVED** [review]
LGTM

**Pull request merged**